package.json

```diff
@@ -35,26 +35,26 @@
   },
   "extensionDependencies": [],
   "dependencies": {
-    "console-stamp": "^0.2.2",
-    "diff": "~3.3.0",
+    "console-stamp": "^0.2.7",
+    "diff": "^3.5.0",
     "json-rpc2": "^1.0.2",
-    "vscode-debug-logger": "^0.0.4",
-    "vscode-debugadapter": "^1.30.0",
-    "vscode-debugprotocol": "^1.11.0",
+    "vscode-debug-logger": "0.0.5",
+    "vscode-debugadapter": "^1.32.1",
+    "vscode-debugprotocol": "^1.32.0",
     "vscode-extension-telemetry": "^0.0.22",
-    "vscode-languageclient": "~4.3.0"
+    "vscode-languageclient": "^5.1.0"
```

Comment:
The last time I changed the major version of the language client, it broke the language server feature :)
Did you get a chance to test the language server?